app/controllers/admin/rooms_controller.rb

```diff
@@ -3,6 +3,7 @@ class RoomsController < Admin::BaseController
     load_and_authorize_resource :conference, find_by: :short_title
     load_and_authorize_resource :venue, through: :conference, singleton: true
     load_and_authorize_resource through: :venue
+    after_action :prepare_unobtrusive_flash
```

Comment:
Filter is used from `gem 'unobtrusive_flash'` documentation. It is used to create a flash message after an ajax call. 
![flash_message](https://cloud.githubusercontent.com/assets/14261624/23960469/146d8696-09ce-11e7-95a6-e6e0b4f3b379.png)